test.js

```diff
@@ -24,4 +24,7 @@ function foo()  {}
 
 function(){}
 function foo (){}
-function foo ()  {}
\ No newline at end of file
+function foo ()  {}
+
+console.log('hello, world!');
+
```

Comment:
ah, I see. :) Was a bit confused because of the words being printed; `'correct EOL'` or similar might be a bit more clear.